contracts/protocol/facets/ProtocolInitializationHandlerFacet.sol

```diff
@@ -161,6 +162,9 @@ contract ProtocolInitializationHandlerFacet is IBosonProtocolInitializationHandl
             (uint256, uint256[], address[])
         );
 
+        // make sure _minResolutionPeriod is less than maxResolutionPeriod
+        require(protocolLimits().maxResolutionPeriod > _minResolutionPeriod, INVALID_RESOLUTION_PERIOD);
```

Comment:
Similar to above, protocolLimits should be cached and I think we can allow the periods to be the same.

Reply:
cached